src/EA.Weee.Web/Areas/AATFReturn/Controllers/ObligatedValuesCopyPasteController.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.Controllers
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using System.Web.Mvc;
+    using EA.Weee.Api.Client;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Core.Shared;
+    using EA.Weee.Requests.AatfReturn;
+    using EA.Weee.Web.Areas.AatfReturn.ViewModels;
+    using EA.Weee.Web.Constant;
+    using EA.Weee.Web.Infrastructure;
+    using EA.Weee.Web.Services;
+    using EA.Weee.Web.Services.Caching;
+
+    public class ObligatedValuesCopyPasteController : AatfReturnBaseController
+    {
+        private readonly BreadcrumbService breadcrumb;
+        private readonly IWeeeCache cache;
+        private readonly Func<IWeeeClient> apiClient;
+        private readonly IPasteProcessor pasteProcessor;
+
+        public ObligatedValuesCopyPasteController(Func<IWeeeClient> apiClient, BreadcrumbService breadcrumb, IWeeeCache cache, IPasteProcessor pasteProcessor)
+        {
+            this.apiClient = apiClient;
+            this.breadcrumb = breadcrumb;
+            this.cache = cache;
+            this.pasteProcessor = pasteProcessor;
+        }
+
+        [HttpGet]
+        public virtual async Task<ActionResult> Index(Guid returnId, Guid aatfId, Guid schemeId)
+        {
+            using (var client = apiClient())
+            {
+                var @return = await client.SendAsync(User.GetAccessToken(), new GetReturn(returnId));
+                var organisationId = @return.ReturnOperatorData.OrganisationId;
+                var viewModel = new ObligatedValuesCopyPasteViewModel()
+                {
+                    AatfId = aatfId,
```

Comment:
Set the organisation id inside here from @return.returnoperatordata.organisationid